linkerd/app/outbound/src/endpoint.rs

```diff
@@ -173,7 +173,7 @@ impl Into<http::client::Settings> for &'_ HttpEndpoint {
 
 impl tap::Inspect for HttpEndpoint {
     fn src_addr<B>(&self, req: &http::Request<B>) -> Option<SocketAddr> {
```

Comment:
[`req` is unused](https://github.com/linkerd/linkerd2-proxy/pull/680/checks?check_run_id=1183684537#step:3:509), changing this to
```suggestion
    fn src_addr<B>(&self, _: &http::Request<B>) -> Option<SocketAddr> {
```
should fix the build.